packages/code-snippet-block/src/CodeSnippetBlock.spec.ct.tsx

```diff
@@ -109,9 +109,9 @@ it('renders code snippet with border radius', () => {
 it('renders code snippet with custom border radius', () => {
     const [CodeSnippetWithStubs] = withAppBridgeBlockStubs(CodeSnippetBlock, {
         blockSettings: {
-            withBorder: true,
-            lineWidth: '2px',
-            lineStyle: 'solid',
+            hasBorder: true,
+            borderWidth: '2px',
+            borderStyle: 'Solid',
```

Comment:
Can be removed, not tested in the test :)

Reply:
I thought I leave it as then it's clearer where to the radius is set when looking at it again. But actually you're right. Removed it 👍